olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -925,7 +925,15 @@ spec:
                       - name: "IMAGE_KUBE_RBAC_PROXY"
                         value: "registry.ci.openshift.org/origin/4.7:kube-rbac-proxy"
                       - name: "IMAGE_KN_PLUGIN_EVENT_SENDER"
-                        value: "registry.ci.openshift.org/knative/release-1.4:client-plugin-event-sender"
```

Comment:
did we never update this before? :sweat_smile:

Reply:
We might have bumped it directly on `release-*` branch a few times. 😰

Reply:
e.g.: https://github.com/openshift-knative/serverless-operator/pull/1884